.github/pull_request_template.md

```diff
@@ -14,6 +14,8 @@ Pick one of the following:
 
 If your change is to providers it can only be (Infra, BugFix or New).
 
+If your change impacts the submission format in Checkbox test reports, ensure that the schema is updated and relevant fields are documented. Follow the instructions in `submission-schema/README.md` to recreate the schema.json file.
```

Comment:
```suggestion
If your change impacts the submission format in Checkbox test reports, ensure that `submission-schema/schema.json` is updated and relevant fields are documented.
```

Reply:
(We don't actually want the schema to be regenerated from examples, it is at this point hand crafted and will continue to be, since we are going to need to start to document keys in there)